src/routes/EResourceViewRoute.js

```diff
@@ -45,6 +45,7 @@ class EResourceViewRoute extends React.Component {
     },
     query: {},
     entitlementsCount: { initialValue: RECORDS_PER_REQUEST },
+    filterPath: { initialValue: 'current' },
```

Comment:
Same sort of comments as the variables in `AgreementViewRoute`. Maybe `packageContentsFilter`, `handleFilterPackageContents` and `onFilterPackageContents`.